aviary/mission/gasp_based/test/test_idle_descent_estimation.py

```diff
@@ -17,7 +18,7 @@ def test_case1(self):
         tol = 1e-5
         input_deck = 'models/large_single_aisle_1/large_single_aisle_1_GwGm.csv'
         aviary_inputs, _ = create_vehicle(input_deck)
-        aviary_inputs.set_val('debug_mode', False)
+        aviary_inputs.set_val('verbosity', Verbosity.QUIET)
```

Comment:
Maybe we should make an issue to add either QUIET or BRIEF to all tests to reduce unnecessary outputs. How does everyone feel about test outputs?

Reply:
I'm not sure if it is possible, but it would be nice if running testflo for the entire repository was QUIET and then failed tests were rerun with verbose.